controllers/appgroup_reconciler.go

```diff
@@ -131,7 +131,7 @@ func (r *ApplicationGroupReconciler) reconcileApplications(l logr.Logger, appGro
 						scc.Templates = append(scc.Templates, t)
 					}
 				}
-				scc.Files = append(scc.Files, appCh.Files...)
+				// scc.Files = append(scc.Files, appCh.Files...)
```

Comment:
Reason this is only commented and not removed entirely?

Reply:
No reason - removing it :)